Core/include/Acts/EventData/TrackStateProxy.hpp

```diff
@@ -776,6 +817,29 @@ class TrackStateProxy {
     m_traj->allocateCalibrated(m_istate, measdim);
   }
 
+  VariableSubspaceHelper<eBoundSize> calibratedVariableSubspace() const {
+    FullProjectorMapping fullMapping = fullProjectorMapping();
+    return VariableSubspaceHelper<eBoundSize>(
+        std::span(fullMapping.begin(), fullMapping.begin() + calibratedSize()));
+  }
+
+  template <std::size_t measdim>
+  FixedSubspaceHelper<eBoundSize, measdim> calibratedFixedSubspace() const {
+    ProjectorMapping<measdim> mapping = projectorMapping();
+    return FixedSubspaceHelper<eBoundSize, measdim>(mapping);
+  }
+
+  template <std::size_t measdim, typename index_t>
+  void setCalibratedSubspace(
```

Comment:
I think one set of methods operates on the pure mapping structure, and the other one on this helper that interprets it.

I think I agree with @goetzgaycken that it might be a better idea to either only operate on the bare types and push using the helper for interpretation to the user, or only expose the helper to the user, and only deal with the raw type internally.